.gitignore

```diff
@@ -16,3 +16,10 @@ minisketch.dll
 
 #Milagro.
 incubator-milagro-crypto-c
+
+#Test files.
+/tests/nim.cfg
+/tests/AllTests.nim
+
+#Nimble temp files.
+/Meros_*.nims
```

Comment:
> These should be cleaned on build.

They should except for when the process is killed. eg `CTRL-C`.

RE: ignore all `*.nims` except `config.nims`, `Meros.nims`.

That should work if it's preferred.

Reply:
I'm indifferent. My main comment was the fact your wildcard only applies to `Meros_`. I'm trying to think of a scenario in which there's a non-`Meros_` temporary nimscript. Even though we have multiple nimscript files, these generated files get their name off Nimble, right? Should be fine since we have one of those.